samples/matter/common/src/bridge/bridge_storage_manager.h

```diff
@@ -183,6 +186,28 @@ class BridgeStorageManager {
 	 */
 	bool RemoveBridgedDeviceEndpointId(uint8_t bridgedDeviceIndex);
 
+	/**
+	 * @brief Load bridged device UniqueID from settings
+	 *
+	 * @param uniqueID address of char array to be filled with loaded data
+	 * @param labelMaxLength maximum size that can be used for UniqueID array
```

Comment:
```suggestion
	 * @param uniqueIDMaxLength maximum size that can be used for UniqueID array
```

Reply:
This method will be removed in a follow-up task.